RHIDP-7661: Correct Fixed Issues Placement in 1.6.1 Release Notes (#1213)
redhat-developer/red-hat-developers-documentation-rhdh: modules/release-notes/ref-release-notes-fixed-issues.adoc

[id="bug-fix-rhidp-6597"]
=== Fixed unreachable links in the Operator installation information page on the OpenShift console

> Previously, the {product-short} Operator details page in the OpenShift web console contained links to some pages that could not be reached outside of the {company-name} network.

Reviewer: Consider using the attributes for all instances of OpenShift to help ensure that we are complying with the approved usage of this product (for instance, _OpenShift_ by itself and _OCP_ are not approved usages).

> With this update, the links in the Operator installation information page have been fixed.

Reviewer: ```suggestion
With this update, the links in the Operator installation information page are fixed.
```

[id="bug-fix-rhidp-7380"]

> === Fixed Profile dropdown showing `Guest` instead of user&#39;s name with OIDC authentication

Reviewer: If Profile is on the UI, then bold it

> Previously, the Profile dropdown in the Global Header showed `Guest` instead of the logged-in user’s display name when logging in using the OIDC authentication.

Reviewer: Need to check if it is rendering correctly, but `user&#39;s` looks a little strange. This issue can likely be avoided by complying with IBM guidelines on possessives.

Reviewer 2: you mean using `username` instead of `user&#39;s name` ?

Reviewer 2: for the curious it looks like whover last updated the release note text in JIRA used special chars like `’` instead of standard single quotes like `'`. Someone's OS was being fancy ;)

> With this update, the Profile dropdown now correctly displays the user&#39;s name by first checking `spec.profile.displayName`, then `metadata.title`, and finally falls back to the name shown on the Profile card in the Settings page if neither is available.

Reviewer: Bold all UI elements and page names
